libr/bin/p/bin_coff.c

```diff
@@ -345,7 +345,10 @@ static RList *_relocs_list(RBin *rbin, struct r_bin_coff_obj *bin, bool patch, u
 					case COFF_REL_I386_REL32:
 						reloc->type = R_BIN_RELOC_32;
 						reloc->additive = 1;
-						ut64 data = sym_vaddr - reloc->vaddr - 4;
+						ut8 offset[4];
+						rbin->iob.read_at (rbin->iob.io, reloc->vaddr, offset, 4);
```

Comment:
check for return value and print a warning and/or assign a default value to `data`?